lib/services/assembly/assembly.ts

```diff
@@ -72,7 +72,7 @@ export class OCCTAssembly {
     traverseAssembly(assembly: XCAFDoc_ShapeTool) {
         const freeShapes = new this.occ.TDF_LabelSequence_1();
         assembly.GetFreeShapes(freeShapes as TDF_LabelSequence);
-        const children = [];
+        const children: (Assembly | Part)[] = [];
```

Comment:
will need to come back to this file at some point. It's quite cool to have assemblies and not so cool to have commented out stuff sitting in the codebase like that :)

Reply:
Yes there was commented out code that I nearly pressed delete on